mne/viz/tests/test_topo.py

```diff
@@ -153,5 +153,7 @@ def test_plot_tfr_topo():
     tfr.plot_topo(baseline=(None, 0), mode='ratio', title='Average power',
                   vmin=0., vmax=14., show=False)
     tfr.plot([4], baseline=(None, 0), mode='ratio', show=False, title='foo')
+    tfr.plot([4], baseline=(None, 0), mode='ratio', show=False, title='foo',
+             imtype='nonuniform')
```

Comment:
very simple "test" - but I am open to suggestions how to make it actually test something more useful